features/init.feature

```diff
@@ -8,6 +8,11 @@ Feature: hub init
     When I successfully run `hub init -g`
     Then the url for "origin" should be "git@github.com:mislav/dotfiles.git"
 
+  Scenario: Initializes a git repo in a new directory with remote
+    When I successfully run `hub init -g new_dir`
+    And I cd to "new_dir"
+    Then the url for "origin" should be "git@github.com:mislav/new_dir.git"
```

Comment:
> I'm a little nitpicky, I know.

Why wouldn't @mislav let go :smiley_cat: ? I added the assertion in https://github.com/github/hub/pull/793.